apps/web-client/src/routes/orders/suppliers/order/[id]/+page.ts

```diff
@@ -0,0 +1,17 @@
+import type { PageLoad } from "./$types";
+import { getPlacedSupplierOrderLines } from "$lib/db/cr-sqlite/suppliers";
+
+export const load: PageLoad = async ({ parent, params }) => {
+	const { ordersDb } = await parent();
+	// console.log({ placedOrders });
+	// const placedOrder = placedOrders?.find((order) => order.id === parseInt(params.id));
```

Comment:
Remove

Reply:
Add/use a separate query to get a placed supplier order meta data. We could potentially add an optional supplierId param to `getPlacedSupplierOrders`?